Solutions/MicrosoftDefenderForEndpoint/Playbooks/Isolate-MDEMachine/readme.md

```diff
@@ -41,4 +47,7 @@ New-AzureAdServiceAppRoleAssignment -ObjectId $MI.ObjectId -PrincipalId $MI.Obje
 ![Incident Trigger](./Isolate-MDEMachine/Isolate-MDEMachine-incident-trigger/images/designerLight.png)
 
 **Alert Trigger**
-![Alert Trigger](./Isolate-MDEMachine/Isolate-MDEMachine-alert-trigger/images/Isolate-MDEMachine_alert.png)>
\ No newline at end of file
+![Alert Trigger](./Isolate-MDEMachine/Isolate-MDEMachine-alert-trigger/images/Isolate-MDEMachine_alert.png)>
+
+**Entity Trigger**
+![Entity Trigger](./Isolate-MDEMachine/Isolate-MDE-Machine-entity-trigger/images/playbookDark.png)>
```

Comment:
For Playbook to add any comment to incident, it should have assigned "Microsoft Sentinel Responder Role."

Kindly add the steps for assigning this role in readme.md file

Reply:
added

Reply:
can we have it approved now as we must publish blog about Tasks - artifacts for it are in SOAR Essential solution.